test/it/fileutils/mv_gnu_test.sh

```
@@ -0,0 +1,29 @@
+Setup() {
+    export TEST_DIR=${MIMIXBOX_IT_ROOT}/mv_gnu
+    mkdir -p ${TEST_DIR}/dst
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)